packages/button/src/utils/getLgIds.ts

```diff
@@ -1,6 +1,8 @@
+import { LgIdString } from '@leafygreen-ui/lib';
+
 export const DEFAULT_LGID_ROOT = 'lg-button';
 
-export const getLgIds = (root: `lg-${string}` = DEFAULT_LGID_ROOT) => {
+export const getLgIds = (root: LgIdString = DEFAULT_LGID_ROOT) => {
```

Comment:
Not necessary here, but it might be worth creating a `GetLgIdsFunction` function type, and maybe a return type?

```ts
type GetLgIdsFunction = (root?: LgIdString) => {
  root: LgIdString,
  ...
}
```